icu4j/main/classes/core/src/com/ibm/icu/util/MutableCodePointTrie.java

```diff
@@ -309,6 +312,15 @@ public void setRange(int start, int end, int value) {
      * Compacts the data and builds an immutable {@link CodePointTrie} according to the parameters.
      * After this, the mutable trie will be empty.
      *
+     * <p>The mutable trie stores 32-bit values until buildImmutable() is called.
+     * If values shorter than 32 bits are to be stored in the immutable trie,
+     * then the upper bits are discarded.
+     * For example, when the mutable trie contains values 0x81, -0x7f, and 0xa581,
+     * and the value width is 8 bits, then each of these is stored as 0x81
+     * and the immutable trie will return that as an unsigned value.
+     * (A builder can make productive temporary use of the upper bits
```

Comment:
(Some implementations may want to make productive temporary ...

Reply:
done